po/de.po

```diff
@@ -2216,15 +2222,31 @@ msgid ""
 "Type \"help (<command>)\" for more information.\n"
 "\n"
 msgstr ""
+"Willkommen im GTGs Entwickerkonsole\n"
```

Comment:
Typo again, should be "in GTGs Entwicklerkonsole"

Reply:
Thanks, fixed.